purchases_ui_flutter/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ environment:
   flutter: '>=3.3.0'
 
 dependencies:
+  purchases_flutter: 6.17.0
```

Comment:
`purchases_ui_flutter` now requires `purchases_flutter` to use the `Offering`.

Reply:
Fixed!

Reply:
Just to confirm, this will make it so we need to publish `purchases_flutter` BEFORE `purchases_ui_flutter`. Are we sequencing those jobs now?

Reply:
Good question let me check